go/vt/mysqlctl/replication.go

```diff
@@ -47,7 +47,7 @@ func WaitForReplicationStart(mysqld MysqlDaemon, replicaStartDeadline int) error
 			return err
 		}
 
-		if status.ReplicationRunning() {
+		if status.Running() {
```

Comment:
Note: this was a bug. It should be using `Running()` and NOT `Healthy()` because it's supposed to wait until we know that we can connect to the source.